TrelloAutotest/Pages/TrelloMainPage.cs

```diff
@@ -73,5 +75,26 @@ public TrelloMainPage VerifyBoardHeaderExists()
             Assert.IsTrue(driver.FindElements(BoardSelectors.BoardHeader).Any());
             return this;
         }
+
+        public TrelloCreateWorkSpacePage DeleteWorkSpace(string workSpaceName)
```

Comment:
I see this method was moved, but what is the reason?

Reply:
Because when I was creating Workspace manually by clicking on the application, the Workspace data page was opening by default (TrelloCreateWorkSpacePage) and the removal process had to be started from that page. After entering Workspace creation through API queries, the Workspace data page no longer opens, so the delete process starts with MainPage.

Reply:
The previous version of the method directed the process to MainPage anyway by clicking the appropriate icon. Now this step is simply skipped.